.azure/pipelines/Phonebook.preview.yml

```diff
@@ -93,9 +93,7 @@ jobs:
         artifact: $(helm_artifact_name)
 
       - task: HelmInstaller@1
-        displayName: 'Install Helm 2.14.2'
-        inputs:
-          helmVersionToInstall: 2.14.2
```

Comment:
What does the task if you remove the version? install the latest? Can this break something?`should we maybe use something like 2.x.x?

Reply:
nope, we updated to the newest and we will automatically update until it breaks again :) 
This way we know if it breaks, and then we can fix the version again.